libraries/core/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,136 @@
+    <string name="menu_logout_prompt_confirm">確認</string>
+    <string name="menu_logout_prompt_cancel">キャンセル</string>
+
+    <string name="sort_dialog_title">並び替え</string>
+    <string name="sort_by_default">デフォルト</string>
+    <string name="sort_by_title">タイトル</string>
+    <string name="sort_by_start_date">開始日</string>
+    <string name="sort_by_end_date">終了日</string>
+    <string name="sort_by_rating">評価</string>
+
+    <string name="filter_dialog_title">絞り込む</string>
+    <string name="filter_dialog_cancel">キャンセル</string>
+    <string name="filter_dialog_confirm">確認</string>
+    <string name="filter_by_current">現在</string>
+    <string name="filter_by_completed">完了</string>
+    <string name="filter_by_on_hold">保留</string>
+    <string name="filter_by_dropped">ドロップ</string>
+    <string name="filter_by_planned">予定</string>
+
+    <string name="series_list_length">%s / %s</string>
+    <string name="series_list_plus_one">+1</string>
+    <string name="series_list_date_range">%s - %s</string>
+    <string name="series_list_unknown">不明</string>
+    <string name="series_list_ongoing">進行中</string>
+    <string name="series_list_try_again">シリーズ％sの更新に失敗しました。もう一度やり直してください…</string>
+    <string name="series_list_delete_title">本当に％sを削除してもよろしいですか？</string>
+    <string name="series_list_delete_confirm">確認</string>
+    <string name="series_list_delete_cancel">キャンセル</string>
+    <string name="series_list_delete_failure">シリーズを削除しようとしましたが、エラーが発生しました</string>
+    <string name="series_list_delete_retry">再試行</string>
+    <string name="series_list_empty">現在のフィルターで見つかったシリーズはありません</string>
+    <string name="series_list_refresh">リフレッシュシリーズ</string>
+    <string name="series_list_refresh_error">シリーズ一覧を更新する際にエラーが発生しました。もう一度やり直してください...</string>
+    <string name="rate_dialog_title">レートシリーズ</string>
+    <string name="series_list_rate_cancel">キャンセル</string>
+    <string name="series_list_rate_confirm">確認</string>
+
+    <string name="series_detail_dash">-</string>
+    <string name="series_detail_progress_title">進捗状況</string>
+    <string name="series_detail_progress_out_of">/ %s</string>
+    <string name="series_detail_status_title">ユーザーシリーズの状況</string>
+    <string name="series_detail_rating">評価</string>
+    <string name="series_detail_confirm">確認</string>
+    <string name="series_detail_failure">シリーズ％sの更新に失敗しました。再試行してください</string>
+
+    <string name="rating_none">評価なし</string>
+
+    <string name="login_header">Kitsuにログイン</string>
+    <string name="login_username">Kitsu eメール</string>
+    <string name="login_password">パスワード</string>
+    <string name="login_login">ログイン</string>
+    <string name="login_forgot_password">パスワードをお忘れですか？こちらから変更出来ます</string>
+    <string name="login_forgot_password_url">https://kitsu.io/password-reset</string>
+    <string name="login_sign_up">こちらからKitsuに登録出来ます</string>
+    <!-- This text should be exactly what part of login_sign_up should be a link -->
+    <string name="login_sign_up_link_target">こちらから</string>
```

Comment:
I've made a change here to show what I think the string should be. It needs to have a matching section in the `login_sign_up` part. Its used just to set up some formatting for the string above.

Reply:
Thanks for changing the translation!